### ISSUE TITLE
Webpack 5 update doesn't allow individual USWDS JS imports

### ISSUE THREAD

**Issue by coreylafferty** [forumone/gesso-uswds#96]
After updating Webpack to version 5 (currently on the 4.x-RC branch), the gulp task errors out on the scripts being imported by scripts.es6.js file.

```
 Error: ModuleNotFoundError: Module not found: Error: Package path ./src/js/components/navigation.js is not exported from package /app/node_modules/uswds (see exports field in /app/node_modules/uswds/package.json)
ModuleNotFoundError: Module not found: Error: Package path ./src/js/components/accordion.js is not exported from package /app/node_modules/uswds (see exports field in /app/node_modules/uswds/package.json)
ModuleNotFoundError: Module not found: Error: Package path ./src/js/components/banner.js is not exported from package /app/node_modules/uswds (see exports field in /app/node_modules/uswds/package.json)
ModuleNotFoundError: Module not found: Error: Package path ./src/js/components/tooltip.js is not exported from package /app/node_modules/uswds (see exports field in /app/node_modules/uswds/package.json)
ModuleNotFoundError: Module not found: Error: Package path ./src/js/components/table.js is not exported from package /app/node_modules/uswds (see exports field in /app/node_modules/uswds/package.json)
```

**Comment by coreylafferty** [comment]
Fixed by #97 

Thanks @kmonahan!

Leaving this open for now as we should be able to revert these changes and go back to individual imports once this USWDS issue is resolved:  https://github.com/uswds/uswds/issues/4194, likely by this PR: https://github.com/uswds/uswds/pull/4196

Note: if we switch back to individual imports, we likely need to include start.js or append this to our domready script calls: `window.uswdsPresent = true;`

**Comment by coreylafferty** [comment]
Created a PR that updates USWDS to 2.12 and reverts these changes since individual imports work again in that version.  #103
